components/molecules/ContributionsEvolution/contributions-evolution-card.tsx

```diff
@@ -127,21 +132,23 @@ export default function ContributionsEvolutionCard(props: Props) {
           {/* chart */}
           <div className="mb-3 grid " style={{ height: "auto" }}>
             <div>
-              <ResponsiveBar
-                data={formattedData}
-                keys={dataTypes as unknown as string[]}
-                indexBy={"startTime"}
-                axisBottom={{ tickSize: 0 }}
-                axisLeft={null}
-                enableLabel={false}
-                enableGridY={false}
-                labelFormat={""}
-                margin={{ top: 0, right: 40, bottom: 30, left: 40 }}
-                motionConfig="stiff"
-                padding={0.5}
-                colors={(d) => colors[d.id as keyof typeof colors]}
-                tooltipLabel={(d) => labels[d.id as keyof typeof labels]}
-              />
+              <ClientOnly>
```

Comment:
This section is wrapped with `<ClientOnly />` so that the rest can be server side rendered and avoid a hydration error.